apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -17,7 +20,18 @@ const AnimationPickerBody = React.createClass({
     onUploadClick: React.PropTypes.func.isRequired
   },
 
+  getInitialState() {
+    return {
+      searchQuery: ''
+    };
+  },
+
+  onSearchQueryChange(value) {
+    this.setState({searchQuery: value});
+  },
+
   render() {
+    var pageOfResults = searchAnimations(this.state.searchQuery);
```

Comment:
Yes.  It was optimistic naming on my part, since I want pagination in the future and could picture `searchAnimations` taking a 'start at index' argument... but I can rename if this is confusing.